rules/check-haveibeenpwned/rule.md

```diff
@@ -0,0 +1,23 @@
+---
+type: rule
+title: Do you check passwords in haveibeenpwned?
+uri: check-haveibeenpwned
+authors:
+  - title: Chris Schultz
+    url: https://www.ssw.com.au/people/chris-schultz
+  - title: Piers Sinclair
+    url: https://www.ssw.com.au/people/piers-sinclair
+related:
+  - password-manager
+created: 2022-03-16T23:33:50.685Z
+guid: 91bcb32d-3ac2-46df-81e1-ccbb3f8512aa
+---
+When an organisation has a security breach, passwords are compromised and there is no visibility of the problem. This problem means that hackers can gain access to people's accounts without anyone realising what has happend! [haveibeenpwned](https://haveibeenpwned.com/) is a website that addresses this problem...
```

Comment:
Fix typo:
from
    > without anyone realising what happend!
to
    without anyone realising what happened!